proxy/pom.xml

```diff
@@ -102,6 +102,21 @@
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
     </dependency>
+    <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+        <version>1.7.5</version>
```

Comment:
we don't normally use a slf4j and logging frameworks are not easy to figure out, was in the intent here to use slf4j and bridging it to log4j?